test/index.spec.ts

```diff
@@ -0,0 +1,213 @@
+import * as isDomNode from "../src/index";
+import assert from "assert";
+import { DOMParser } from "@xmldom/xmldom";
```

Comment:
A reasonable first choice, would of course be nice in the future to check other implementations like the one from Deno, browsers or jsdom.

Reply:
You mean to have a suite of tests that tests against objects created by those other frameworks?

Reply:
I've added a suite for `jsdom`, but I'm not sure how I'd add one for Deno or browsers. At this point, I feel like that would just be testing that those work correctly, not this code. Testing two libraries ensures that we are "doing the right thing™" and after that I feel like the standards should be enough to keep everyone on the same page. LMK what you think.

Reply:
Awesome, this was more like "thinking out loud about ideas for the future".
And adding a second one is already showing that this is not just about the xmldom package.